wacruit/src/admin/formatters.py

```
@@ -38,3 +38,11 @@ def timeline_category_formatter(
 ) -> str | None:
     category = getattr(model, "category")
     return category and category.title
+
+
+def project_urls_formatter(model: type[DeclarativeBase], attribute: str):
```

Ruff (0.13.1)

Unused function argument: `attribute`

(ARG001)